unidist/core/backends/mpi/core/serialization.py

```diff
@@ -21,6 +22,15 @@
 import msgpack
 import gc  # msgpack optimization
 
+# Pickle 5 protocol compatible types check
+compatible_modules = ("pandas", "numpy")
```

Comment:
Shouldn't we highlight in our docs that a fast serialization of complex datatypes is supported only for `pandas.DataFrame` and `np.ndarray`?

Reply:
Good point! However, I don`t really know where is the best place for such performance notes